src/AppBridgeBlock.ts

```diff
@@ -401,7 +396,7 @@ export class AppBridgeBlock {
         };
     }
 
-    private mapAssetApiToFrontifyAsset(asset: AssetApi): FrontifyAsset {
+    private mapAssetApiToFrontifyAsset(asset: AssetApi): Asset {
```

Comment:
And I think we should update also the name of the methods:
```suggestion
    private mapAssetApiToAsset(asset: AssetApi): Asset {
```